CMakeLists.txt

```diff
@@ -4,6 +4,8 @@ if(MSVC)
 endif()
 project(Cppcheck VERSION 2.16.99 LANGUAGES CXX)
 
+include(cmake/options.cmake)
```

Comment:
Moved to the top otherwise the default values for the options will not be applied yet.